lib/gui/app/pages/main/templates/main.tpl.html

```diff
@@ -106,6 +106,12 @@
             <span ng-bind="flash.getProgressButtonLabel()"></span>
         </progress-button>
 
+        <button class="button button-link button-abort-write"
```

Comment:
I wonder if it'd be worth adding a tooltip to this button, to make its purpose more obvious?

Reply:
Yeah, @Shou @konmouz wdyt?